Xamarin.Forms.Core/ExpanderOrientation.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Xamarin.Forms
+{
+	public enum ExpanderOrientation
+	{
+		TopToBottom,
+		BottomToTop,
+		LeadingToTrailing,
+		TrailingToLeading
```

Comment:
Used Leading/Trailing because of "RTL"

Reply:
StartToEnd ? Like we have LayoutOptions

Reply:
> StartToEnd ? Like we have LayoutOptions

I thought about it. But Start & End are used both for Vertical and Horizontal LayoutOptions.

Then it would make sense to rename all options in this enum
VerticalStartToEnd
VerticalEndToStart
HorizontalStartToEnd
HorizontalEndToStart